src/app/components/ModeRules.js

```diff
@@ -0,0 +1,26 @@
+const rulesTexts = {
+  people:
+    'You have one minute (1m) to answer as many questions as possible. During the game on each question you need to select who from Star Wars is showed on the left (Jar Jar Binks right now) from available options.',
+  vehicles:
+    'You have one minute (1m) to answer as many questions as possible. During the game on each question you need to select which vehicle from Star Wars is showed on the left.',
+  starships:
+    'You have one minute (1m) to answer as many questions as possible. During the game on each question you need to select which starship from Star Wars is showed on the left.',
+};
+
+const ModeRules = (mode) => {
+  const rulesElement = document.createElement('div');
+  rulesElement.classList.add('rules');
+  const rulesHeader = document.createElement('header');
+  rulesHeader.classList.add('rules-header');
+  const rulesIcon = document.createElement('span');
+  rulesIcon.classList.add('rules-icon', 'fa', 'fa-graduation-cap');
+  const rulesTextElement = document.createElement('div');
+  rulesTextElement.classList.add('rules-text');
```

Comment:
Paulina said that we should declarate all variables first, then methods like classList.add

Reply:
Sure, I'll change that.